mpl_volume_viewer.py

```diff
@@ -47,7 +47,7 @@ def _normalize_fig_axes(fig, axes):
 class SliceViewer:
     def __init__(self, volume, spacing=None, cmap=plt.cm.gray,
                  points=None, pts_depth=2, pts_color='red',
-                 labels=None, labels_cmap='random', multichannel=False,
+                 labels=None, labels_cmap=None, multichannel=False,
```

Comment:
This is to display segmentations and should remain random by default, I think.

Reply:
ah, ok. I was confused because this argument was not being used.